src/nodes/probit.jl

```diff
@@ -0,0 +1,45 @@
+using StatsFuns: normcdf
+
+export Probit, ProbitMeta
+
+struct Probit end
+
+struct ProbitMeta
+    p :: Int
+end
+
+getp(meta::ProbitMeta) = meta.p
+
+ProbitMeta(; p = 32) = ProbitMeta(p)
+
+@node Probit Stochastic [ out, in ]
+
+default_meta(::Type{ Probit }) = ProbitMeta(32)
+
+default_functional_dependencies_pipeline(::Type{ <: Probit }) = RequireInboundFunctionalDependencies((2, ), (vague(NormalMeanPrecision), ))
+
+default_interface_local_constraint(::Type{ <: Probit }, edge::Val{ :in })  = MomentMatching()
+default_interface_local_constraint(::Type{ <: Probit }, edge::Val{ :out }) = Marginalisation()
```

Comment:
@bartvanerp Review this and I will merge

Reply:
The changes look good. I added a final update rule to deal with missing data.

Two problems occurred for the tests:
1) It is not clear how to test for Missing inputs
2) The moment matching constraint is not implemented for the test macro I believe

Reply:
@bartvanerp To address your questions

1) Lets not implement `missing` rules for now, we don't have them for any node yet (only defined in notebooks as an example), I'm still not sure how it should properly work, for now user can define `missing` rules in their notebooks to make it work, but mathematical meaning of such rules has to be discussed yet.
2) yeap, we just forgot to `export MomentMatching`, that is fixed